### PR TITLE
docs: add storageclass to complete-postgres-manifest example

### PR DESCRIPTION
**Pull request by knan-nrk**
Small documentation update.

### CODE REVIEW

File: manifests/complete-postgres-manifest.yaml [hunk 1]
```diff
@@ -8,9 +8,21 @@ spec:
   - name: date
     image: busybox
     command: [ "/bin/date" ]
+  sidecars:
```
Comment (FxKu) [inline manifests/complete-postgres-manifest.yaml]: We already have a sidecar example in the manifest.
Also keep in mind, that many users start testing the operator with the provided manifests.
Therefore, dummy code should be in comments.

Comment (knan-nrk) [reply manifests/complete-postgres-manifest.yaml]: In that case, complete-postgres-manifest needs more things commented out, like the replication slots. And init_containers are not sidecars as such, IMO.

Comment (FxKu) [reply manifests/complete-postgres-manifest.yaml]: I was referring to [this](https://github.com/zalando/postgres-operator/blob/master/manifests/complete-postgres-manifest.yaml#L75).

Comment (knan-nrk) [reply manifests/complete-postgres-manifest.yaml]: Oh, right, never mind me then. I seem to be slightly behind the times. :)

I'll drop the sidecar bit.

**Comment by knan-nrk** [comment]
Updated. :)

**Comment by sdudoladov** [comment]
:+1:

**Comment by FxKu** [comment]
:+1:

**Pull request merged**